TUM-Dev/gocast
# Feature/add lecture time change

### Motivation and Context
No possibility to change date and time of a lecture.

### Description
- Added Timing Fields to Lectures
- Added Logs to changeSet
- Added Computed Properties

### Steps for Testing

Prerequisites:
- 1 Lecturer

1. Log in
2. Navigate to a Course Management Page
3. Change Date Time of a Lecture
4. Reload Page
5. Check if Time Date is still changed
6. Check if other settings can be changed too.

### Screenshots

https://github.com/TUM-Dev/gocast/assets/17506411/8aeb079d-7fa6-4130-b1b3-127d210f9fa1

![Bildschirmfoto 2023-09-27 um 22 03 55](https://github.com/TUM-Dev/gocast/assets/17506411/7c53bcfc-223b-4ee0-9b39-f8a72c5b3d3b)
